### PR TITLE
docs: Update nuget upgrade steps

### PR DESCRIPTION
**Pull request by jeromelaban**

### CODE REVIEW

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.1

File: doc/articles/upgrading-nuget-packages.md [hunk 1]
```diff
@@ -30,12 +30,28 @@ It typically looks similar to this:
 ```
 
 > [!IMPORTANT]
-> At this time the NuGet Manager in Visual Studio 2022 does not parse or manage Sdks provided by NuGet. If you would like to see this feature added, please be sure to provide your [feedback or upvote this issue](https://github.com/NuGet/Home/issues/13127).
+> In Visual Studio 2022 once the Uno Version is updated, you'll need to close/reopen the solution or restart Visual Studio for the change to take effect.
+>
+> At this time the NuGet package Manager does not parse or manage Sdks provided by NuGet. If you would like to see this feature added, please be sure to provide your [feedback or upvote this issue](https://github.com/NuGet/Home/issues/13127).
 
-To update the version, pick the latest stable build from the [Uno.WinUI](https://www.nuget.org/packages/Uno.WinUI) package with either:
+To find the version to updatet to, pick the latest stable build from the [Uno.WinUI](https://www.nuget.org/packages/Uno.WinUI) package with either:
```
Comment (Youssef1313) [inline doc/articles/upgrading-nuget-packages.md:37]:
```suggestion
To find the version to update to, pick the latest stable build from the [Uno.WinUI](https://www.nuget.org/packages/Uno.WinUI) package with either:
```

File: doc/articles/upgrading-nuget-packages.md [hunk 2]
```diff
@@ -30,12 +30,28 @@ It typically looks similar to this:
 ```
 
 > [!IMPORTANT]
-> At this time the NuGet Manager in Visual Studio 2022 does not parse or manage Sdks provided by NuGet. If you would like to see this feature added, please be sure to provide your [feedback or upvote this issue](https://github.com/NuGet/Home/issues/13127).
+> In Visual Studio 2022 once the Uno Version is updated, you'll need to close/reopen the solution or restart Visual Studio for the change to take effect.
```
Comment (Youssef1313) [inline doc/articles/upgrading-nuget-packages.md:33]:
```suggestion
> In Visual Studio 2022, once the Uno Version is updated, you'll need to close and reopen the solution or restart Visual Studio for the change to take effect.
```

File: doc/articles/upgrading-nuget-packages.md [hunk 3]
```diff
@@ -30,12 +30,28 @@ It typically looks similar to this:
...
+>
+> At this time the NuGet package Manager does not parse or manage Sdks provided by NuGet. If you would like to see this feature added, please be sure to provide your [feedback or upvote this issue](https://github.com/NuGet/Home/issues/13127).
```
Comment (Youssef1313) [inline doc/articles/upgrading-nuget-packages.md:35]:
```suggestion
> At this time, the NuGet package Manager does not parse or manage Sdks provided by NuGet. If you would like to see this feature added, please be sure to provide your [feedback or upvote this issue](https://github.com/NuGet/Home/issues/13127).
```

File: doc/articles/upgrading-nuget-packages.md [hunk 4]
```diff
@@ -30,12 +30,28 @@ It typically looks similar to this:
...
 
-To update the version, pick the latest stable build from the [Uno.WinUI](https://www.nuget.org/packages/Uno.WinUI) package with either:
+To find the version to updatet to, pick the latest stable build from the [Uno.WinUI](https://www.nuget.org/packages/Uno.WinUI) package with either:
 
-- [Uno.WinUI in Nuget Package Explorer](https://nuget.info)
+- When using Visual Studio 2022, use the NuGet Package Manager
+- Use [`dotnet outdated`](https://github.com/dotnet-outdated/dotnet-outdated):
+  - Install the tool using:
+
+    ```xaml
```
Comment (Youssef1313) [inline doc/articles/upgrading-nuget-packages.md:43]:
~~~suggestion
    ```dotnetcli
~~~

File: doc/articles/upgrading-nuget-packages.md [hunk 5]
```diff
@@ -30,12 +30,28 @@ It typically looks similar to this:
...
+    dotnet tool install --global dotnet-outdated-tool
+    ```
+
+  - Then, at the root of the solution, run the tool with:
+
+    ```xaml
```
Comment (Youssef1313) [inline doc/articles/upgrading-nuget-packages.md:49]:
~~~suggestion
    ```dotnetcli
~~~

**Review by Youssef1313 — APPROVED** [review]

**Pull request merged**